charts/fleet/charts/gitjob/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 0.1.15
-description: Controller that run jobs based on git events
 name: gitjob
-version: 0.1.15
+description: Controller that run jobs based on git events
```

Comment:
The latest release is 0.1.17(@jiaqiluo Added cve fix)

Reply:
Also you might need to copy the chart over. There are other changes besides image and version bump.

Reply:
I think we should look at the GitJob PR first, actually: https://github.com/rancher/fleet/pull/441#discussion_r677794034

Reply:
Fixed